### DISCUSSION TITLE
Shelly plus 1 ds3231 not recognized

### DISCUSSION THREAD

**Discussion by 13** [arendst/Tasmota#22189] [Q&A] [answered]
I soldered directly to the pinouts, but ds3231 is not detected. Custom solo1 with enabled rtc and ds3231. On a esp32 everything works as expected.

Is there a solo1 bug in the software?

SDA GPIO19
SCL GPIO25 (desoldered button and pull-up resistor)

**Comment by sfromis** [comment, marked as answer]
Did you do any verification of the modification and build working as you want?

Can you get a simple switch/button type input to work for the gpios? Does `I2CScan` show I2C address `0x68` for the chip? Does `I2CDriver` show number 26 for having the driver included? Did you make sure to have good connections on all 4 wires needed? You cannot assume that a wire is not invisibly broken.
